### PR TITLE
AT-1490 paywall schema

### PR DESCRIPTION
**Pull request by quarterto**

### CODE REVIEW

File: server/lib/article/assemble.js [hunk 1]
```diff
@@ -18,14 +18,11 @@ const getCSS = require('./css');
 const environmentOptions = require('./environment-options');
 const handlebars = require('../handlebars');
 
-const getAuthors = data => {
-	const authors = (data.metadata || [])
```
Comment (quarterto) [inline server/lib/article/assemble.js]: A wild `metadata` appeared!

Matt used CAPI V2!

It's super effective!

File: server/lib/article/assemble.js [hunk 2]
```diff
@@ -18,14 +18,11 @@ const getCSS = require('./css');
...
-		.filter(item => !!(item.taxonomy && item.taxonomy === 'authors'))
-		.map(item => item.prefLabel);
-
+const getAuthors = article =>
 	// Somtimes there are no authors in the taxonomy. It's very sad but it's true.
-	return authors.length ? authors.join(', ') : (data.byline || '').replace(/^by\s+/i, '');
-};
+	article.authorConcepts.length
+		? article.authorConcepts.join(', ')
+		: (article.byline || '').replace(/^by\s+/i, '');
```
Comment (rowanbeentje) [inline server/lib/article/assemble.js]: From t'app, editorial are quite particular about the order of authors in the byline, and the concepts aren't sorted the same way - so probably best to byline first, concepts after...

Comment (rowanbeentje) [reply server/lib/article/assemble.js]: (we have a bug in the app backlog for this now, where we'll likely need to show byline and string replace in the author concept links 😱 )

Comment (quarterto) [reply server/lib/article/assemble.js]: this isn't shown to users. i don't think it's shown anywhere, it's just required for a valid schema.org `NewsArticle`. the actual displayed byline does exactly that replace

Comment (rowanbeentje) [reply server/lib/article/assemble.js]: Fair enough!  Looks like there's an error if there's no authorConcepts but approachwise looks good then.

**Pull request merged**